release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -44,6 +44,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-10-azure-gen2-vm"]
+==== Support for Azure generation 2 VMs
+
+{product-title} 4.10 introduces support for provisioning VMs with disk sizes that exceed 2000 GB by using a VM with Azure generation 2 support. For more information, see link:https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2#generation-2-vm-sizes[Support for generation 2 VMs on Azure].
```

Comment:
Do you want to link to the top of that page? 
https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2